docs/variables/sender.md

```diff
@@ -4,118 +4,220 @@ id: sender
+#### Example Output
+
+* `$(sender.accountsince)`
+
+    ```
+    Mon, 06 Apr 2015 23:09:03 UTC
+    ```
 
 ## $(sender.display_name)
 
-Returns the uppercased version, or internationalized version of the senders' username.
+Returns the same as `$(sender)`, the uppercase version, or internationalized version of the sender's username.
+
+#### Parameters
+
+This variable does not take any parameters.
 
-```
-Aiden
-```
+#### Example Output
+
+* `$(sender.display_name)`
+
+    ```
+    Aiden
+    ```
 
 ## $(sender.followers)
 
-Returns the number of followers the sender has.
+Returns the current number of followers of the sender.
+
+#### Parameters
 
-```
-5304
-```
+This variable does not take any parameters.
+
+#### Example Output
+
+* `$(sender.followers)`
+
+    ```
+    5305
+    ```
 
 ## $(sender.game)
 
 Returns the current Twitch directory the sender is under.
 
-```
-Just Chatting
-```
+#### Parameters
+
+This variable does not take any parameters.
+
+#### Example Output
+
+* `$(sender.game)`
+
+    ```
+    Just Chatting
+    ```
 
 ## $(sender.login)
 
-Returns the same as `$(sender)`, the lowercase sender username (otherwise known as "login").
+Returns the same as `$(sendername)`, the lowercase version of the sender's username (otherwise known as *login*).
 
-```
-aiden
-```
+#### Parameters
+
+This variable does not take any parameters.
+
+#### Example Output
+
+* `$(sender.login)`
+
+    ```
+    aiden
+    ```
 
 ## $(sender.provider)
 
-The platform that the message was sent from.
+Returns the platform the sender sent the message from (i.e. twitch).
+
+#### Parameters
+
+This variable does not take any parameters.
+
+#### Example Output
 
-```
-twitch
-```
+* `$(sender.provider)`
+
+    ```
+    twitch
+    ```
 
 ## $(sender.provider_id)
 
-The users' platform ID that the message was sent from.
+Returns the platform ID (e.g. the Twitch ID) of the sender.
+
+#### Parameters
+
+This variable does not take any parameters.
+
+#### Example Output
 
-```
-87763385
-```
+* `$(sender.provider_id)`
+
+    ```
+    87763385
+    ```
 
 ## $(sender.title)
 
-Returns the current title on the senders' Twitch channel.
+Returns the current stream title of the sender's Twitch channel.
+
+#### Parameters
+
+This variable does not take any parameters.
+
+#### Example Output
+
+* `$(sender.title)`
+
+    ```
+    Welcome to my stream!
+    ```
+
+## $(sender.uptime)
+
+Returns the time that has passed since the sender's stream has gone live.
+
+#### Parameters
 
-```
-Welcome to my stream!
-```
+This variable does not take any parameters.
+
+#### Example Output
+
+* `$(sender.uptime)`
+
+    ```
+    3 hours and 25 minutes
+    ```
+
+#### Error Output
+
+* In case the sender's stream is currently offline, returns the following:
+
+    ```
+    [Error: Stream is offline.]
+    ```
 
 ## $(sender.viewers)
 
-Returns the current number of viewers the sender has on their stream.
+Returns the current number of viewers watching the stream of the sender.
```

Comment:
Would it make sense to say `current number of viewers watching the senders' stream?`

Reply:
Yeah that makes sense and looks better. 👍

I'll also change it for **user** and **broadcaster** to be consistent.